.github/workflows/build-push-docker-images.yaml

```diff
@@ -0,0 +1,183 @@
+name: Build-Push-Docker-Image
+
+on:
+  push:
+    branches:
+      - build-push-docker-images
```

Comment:
reminder to make this trigger on tag or push-to-master or something (seems to be being used to trigger workflow runs over at https://github.com/thierrymarianne/contrib-rust-verification-tools/actions )

Reply:
Indeed, the image build remaining quite resource-demanding, I didn't which trigger would make more sense to you folks.